src/types.ts

```diff
@@ -113,3 +113,13 @@ export type Dappeteer = {
     rejectDialog: () => Promise<void>;
   };
 };
+
+export enum AddTokenStatus {
+  SUCCESS = "addTokenResultSuccess",
+  FAIL = "addTokenResultFail",
+}
+
+export enum AddNetworkStatus {
+  SUCCESS = "addNetworkResultSuccess",
+  FAIL = "addNetworkResultFail",
+}
```

Comment:
Nice idea, but think that boolean is simpler solution for it

cc @Lykhoyda @irubido @mpetrunic

Reply:
Agreed, I was just taking what was there though, and didn't change anything but to declare this in the types. Should I go for a boolean? That's a breaking change in the interface then.

Reply:
ok, I removed these and used plain booleans :)